src/data/global.json

```diff
@@ -253,6 +253,10 @@
                             "title": "Privacy Statement",
```

Comment:
Capitalization is inconsistent in this section:

![image](https://user-images.githubusercontent.com/66998419/156799643-c653e7c2-4f15-460a-8e98-7df0d49bb637.png)

Suggest to change "Privacy Statement" to "Privacy statement" to match other items in the list.

Reply:
Yes, indeed, I think your suggestion is the way to go. Thanks.

Reply:
Fixed in https://github.com/corona-warn-app/cwa-website/pull/2508/commits/4799ce6137f4737239b709eada61369ca3d85ee8, thanks!